lib/rubocop/cop/rspec/context_wording.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # `context` block descriptions should start with 'when', or 'with'.
+      #
+      # @see https://github.com/reachlocal/rspec-style-guide#context-descriptions
+      # @see http://www.betterspecs.org/#contexts
+      #
+      # @example `Prefixes` configuration option, defaults: 'when', and 'with'
+      #   Prefixes:
+      #     - when
+      #     - with
+      #     - without
+      #
+      # @example
+      #   # bad
+      #   context 'the display name not present' do
+      #     ...
+      #   end
+      #
+      #   # good
+      #   context 'when the display name is not present' do
+      #     ...
+      #   end
+      class ContextWording < Cop
+        MSG = 'Start context description with %s.'.freeze
+        DEFAULT_PREFIXES = %w[when with].freeze
```

Comment:
@Darhazer Regarding "without"

betterspecs state it as:
> start its description with "when" or "with"

and rspec-style-guide as:
> should always start with 'when'

So I don't see a reason to add it on our side.